### ISSUE TITLE
Train loss is decreaing while validation loss is increasing

### ISSUE THREAD

**Issue by prakashjayy** [kuangliu/pytorch-retinanet#29]
Hi,

The train loss is decreasing with each epoch and validation loss is increasing and becoming stagnant after sometime. 

Can u put up some results on VOC datasets. so that we can cross check ?

Thanks
